src/frontend/hooks/useLanddropdownFelt.tsx

```diff
@@ -18,13 +18,18 @@ const useLanddropdownFelt = (
         skalFeltetVises: avhengigheter => {
             return avhengigheter && avhengigheter.skalFeltetVises;
         },
-        valideringsfunksjon: (felt: FeltState<Alpha3Code | ''>) => {
+        valideringsfunksjon: (felt: FeltState<Alpha3Code | ''>, avhengigheter) => {
             return felt.verdi !== ''
                 ? ok(felt)
-                : feil(felt, <SpråkTekst id={feilmeldingSpråkId} />);
+                : feil(
+                      felt,
+                      avhengigheter?.feilmeldingSpråkId !== '' ? (
+                          <SpråkTekst id={avhengigheter?.feilmeldingSpråkId} />
+                      ) : null
```

Comment:
Burde dette være en tom string?

Reply:
Tja, siden den vil ha en reactnode inn her, og standard for å ikke rendre noe i en ractkomponent er å returne null så brukte jeg null her også, men kan godt endre til '' hvis det er tydeligere

Reply:
Ja, tror det var vi som endret den en gang i tiden til å være en reactnode for å kunne ta inn en språkkomponent 🤔 synes tom streng er tydeligere når det er snakk om rendring av feilmelding